software/utils/rpi_config/bootstrap.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+# Fail if any command fails
+set -e
+set -o pipefail
+
+echo "==============================================================================="
+echo "=================== RespiraWorks Ventilator bootstrapper ======================"
+echo "==============================================================================="
+echo " "
+echo "  -- clones the repository"
+echo "  -- configures Raspberry Pi:"
+echo "     * no screensaver or screen blanking"
+echo "     * no splash on bootup"
+echo "     * serial interface enabled for GUI<->controller comms"
+echo "  -- udev rule added for nucleo, enabling flashing of firmware"
+echo "  -- guake console accessible at all times using [F12]"
+echo "  -- desktop shortcuts for ventilator GUI, updater, debug interface"
+echo "  -- RW-theme background"
+echo "  -- installs toolchains and dependencies for building controller and GUI"
```

Comment:
Missing matplotlib (`pip3 install matplotlib`) to allow running controller_debug, not sure where that should live (controller.sh --install doesn't seem like the right place)